config/__init__.py

```diff
@@ -22,9 +22,11 @@
 RATELIMIT_STRATEGY = 'moving-window'
 
 DEBUG = False
+SESSION_COOKIE_SECURE = True
 
 if not PRODUCTION:
```

Comment:
I think until https://github.com/webcompat/webcompat.com/issues/1303 is done, having a secure cookie on non-HTTPS site won't do what we want. AFAIU, the browser won't send the cookie unless it's talking to a TLS endpoint:

https://tools.ietf.org/html/rfc6265#section-4.1.2.5